sopel/builtins/safety.py

```diff
@@ -360,7 +360,7 @@ def virustotal_lookup(
     }
     bot.memory[SAFETY_CACHE_KEY][url] = result
     if len(bot.memory[SAFETY_CACHE_KEY]) >= (2 * CACHE_LIMIT):
-        _clean_cache(bot)
+        _clean_cache(bot)  # type: ignore[arg-type]
```

Comment:
Does this happen on `master` without the rest of this patch? We should pull it out into a separate PR if so.

Reply:
No, because a. the function _clean_cache is not altered, and b. there is no type check until this.

That being said, as you can see it works at runtime, and it will be better once we remove the SopelWrapper class, because the PluginJob's call signature requires a Sopel instance instead.

Reply:
Oof, I've had a bug in my mental model for years: I thought SopelWrapper was a subclass of Sopel. Whoops! 😂